tests/apps/actordotnet/Controller.cs

```diff
@@ -0,0 +1,44 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace DaprDemoActor
+{
+  using Dapr.Actors;
+  using Dapr.Actors.Client;
+  using Microsoft.AspNetCore.Mvc;
+  using System.Threading.Tasks;
+  using System.IO;
+
+  [ApiController]
+  [Route("/")]
+  public class Controller : ControllerBase
+  {
+    [HttpPost("incrementAndGet/{actorType}/{actorId}")]
+    public async Task<ActionResult<int>> IncrementAndGetAsync([FromRoute] string actorType, [FromRoute] string actorId)
+    {
+      var proxy = ActorProxy.Create(new ActorId(actorId), actorType);
+      return await proxy.InvokeAsync<int, int>("IncrementAndGetAsync", 1);
+    }
+
+    [HttpPost("carFromJSON/{actorType}/{actorId}")]
+    public async Task<ActionResult<Car>> CarFromJSONAsync([FromRoute] string actorType, [FromRoute] string actorId)
+    {
+      using (var ms = new MemoryStream(2048))
+      {
+        await Request.Body.CopyToAsync(ms);
+        string json = System.Text.Encoding.UTF8.GetString(ms.ToArray());
```

Comment:
If you want to make this more idiomatic:

```C#
using var reader = new StreamReader(Request.Body, Encoding.UTF8);
var text = await reader.ReadToEndAsync();
```

Reply:
Done.